Formula/meta-package-manager.rb

```diff
@@ -0,0 +1,417 @@
+class MetaPackageManager < Formula
+  include Language::Python::Virtualenv
+
+  desc "🎁 wraps all package managers with a unifying CLI"
+  homepage "https://github.com/kdeldycke/meta-package-manager"
+  url "https://files.pythonhosted.org/packages/f5/28/5235fca899ead8059fbb35dd4bd786c1dbd4e1d0d65ae2d136540681a60c/meta_package_manager-5.13.0.tar.gz"
+  sha256 "8452718bbcec58dac5fd27691b1be621fcc0d25eb54cb3d3a7db89cfd7f2b1d3"
+  license "GPL-2.0-only"
```

Comment:
This is actually [`GPL-2.0-or-later`](https://github.com/kdeldycke/meta-package-manager/blob/main/license#L299).

```suggestion
  license "GPL-2.0-or-later"
```

Reply:
Good point. The irony here is I'm the one who choose that license first! 😅